tests/phpunit/Logger/JsonLoggerTest.php

```diff
@@ -210,12 +214,13 @@ public function metricsProvider(): iterable
 
         yield 'uncovered mutations' => [
             false,
-            $this->createUncoveredMetricsCalculator(),
+            new MetricsCalculator(2),
+            $this->createUncoveredResultsCollector(),
             [
                 'stats' => [
-                    'totalMutantsCount' => 1,
+                    'totalMutantsCount' => 0,
                     'killedCount' => 0,
-                    'notCoveredCount' => 1,
```

Comment:
could you please explain why the stats was changed?

Reply:
I got it back.